test/client/metadataApiRetrieve.test.ts

```diff
@@ -68,27 +69,24 @@ describe('MetadataApiRetrieve', async () => {
         }
       });
 
-      it('should call retrieve with given options', async () => {
+      it('should call retrieve with for unpackaged data', async () => {
```

Comment:
I think we should have tests for:

1. retrieving unpackaged
2. retrieving a package
3. retrieving multiple packages
4. retrieving a package with unpackaged

And we should verify the component paths are correct.  Also please make sure the names of the tests make sense without silly grammar errors like "with with".  :)